src/target.c

```diff
@@ -729,3 +778,11 @@ unsupported_target_is_open( const struct stumpless_target *target ) {
   raise_target_unsupported( L10N_UNSUPPORTED_TARGET_IS_OPEN_ERROR_MESSAGE );
   return 0;
 }
+
+int 
+stumpless_entry_to_errorstream( const char *buffer, int buffer_len ){
```

Comment:
With the above streamlining, this may not need to be a separate function as it will only be called once. If you do decide to keep it, change it to a private function (move to the `include/private/target.h` header) and rename it to something like `write_buffer_to_error_stream`.